app/templates/components/public/ticket-list.hbs

```diff
@@ -88,12 +88,19 @@
                     @id={{concat ticket.id "_quantity"}}
                     @value={{ticket.orderQuantity}} />
                   <i class="dropdown icon"></i>
-                  <div class="default text">0</div>
+                  <div class="default text">{{if (eq this.tickets.length 1) (if (gt ticket.minOrder 1) ticket.minOrder 1) 0}}</div>
```

Comment:
Why the need of this logic now?

Reply:
bcz we need to show if no of ticketes type is '1' and minOrer is 0 then it should be 1 else minOrder 
we can improve this by 
```handlebars
{{if (eq this.tickets.length 1) (max ticket.minOrder 1) 0}}
```